modules/ovn-kubernetes-architecture-con.adoc

```diff
@@ -23,5 +25,4 @@ It translates the logical network configuration in terms of conventional network
 The OVN southbound database has physical and logical representations of the network and binding tables that link them together. Every node in the cluster is represented in the southbound database, and you can see the ports that are connected to it.
```

Comment:
this might be a bit confusing without mentioning what local and remote are.
local node = node on which SBDB is on
remote node = all other nodes in the cluster.